docs/wave/getting_started.rst

```diff
@@ -14,28 +14,81 @@ Before installing Wave, ensure you have the following prerequisites:
 4. A compatible AMD GPU with ROCm support (MI250, MI300, etc.)
 5. Rust 1.70 or later
 
-Installation
--------------
+Quickstart
+--------------
 
-1. Install PyTorch with ROCm support:
+Wave supports both end users (who want to use Wave kernels in their ML workflows) and developers (who want to contribute to or extend Wave). Follow the relevant guide below to get started quickly.
 
-   .. code-block:: bash
+For Users
+~~~~~~~~~
+
+1. **Install ROCm PyTorch Dependencies**
+
+   Before installing Wave, ensure you have the appropriate ROCm-enabled PyTorch dependencies:
 
+   .. code-block:: bash
+   
       pip install -r pytorch-rocm-requirements.txt
 
-2. Install Rust:
+2. **Install Rust**
 
    .. code-block:: bash
 
       curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 
-3. Install Wave and its dependencies:
+3. **Install Wave**
+
+   You can then install Wave and its dependencies using pip:
+
+   .. code-block:: bash
+
+      pip install wave-lang
+
+For Developers
+~~~~~~~~~
+
+1. **Clone the Repository**
 
    .. code-block:: bash
 
-      pip install -r requirements.txt
-      pip install -r requirements-wave-runtime.txt
+      git clone https://github.com/iree-org/wave.git
+      cd wave
+
+2. **Install Development Dependencies**
+
+   It's recommended to use a virtual environment:
```

Comment:
Similar to above, I would put the first step as install Rust and then all the python installs.